specs/auditbeat.spec.yml

```diff
@@ -37,6 +37,10 @@ inputs:
         - "gc_percent=${AUDITBEAT_GOGC:100}"
         - "-E"
         - "auditbeat.config.modules.enabled=false"
+        - "-E"
+        - "logging.event_data.to_stderr=true"
+        - "-E"
+        - "logging.event_data.to_files=false"
```

Comment:
I'm fine with removing the file, but if we keep the file I rather have it consistent with the feature. I know agentbeat won't be reverted, but if it would, then my PR would still work.

Should I remove those files as part of my PR or do you prefer creating a new PR just to remove them? I can create the PR.

Reply:
Yeah its okay to leave the changes. We should work to remove them, in a seperate PR. Just make sure this change is on the `agentbeat.spec.yml` as well.

Reply:
> Yeah its okay to leave the changes. We should work to remove them, in a seperate PR. Just make sure this change is on the agentbeat.spec.yml as well.

It is, the integration test ensures that. Without those changes/CLI arguments the event log file is not created.